app/forms/user.py

```diff
@@ -85,6 +75,18 @@ def __call__(self, form, field):
             raise ValidationError(message % dict(min=min, max=max, length=length))
 
 
+class LoginForm(RedirectForm):
```

Comment:
I moved this down here because it uses `UsernameLength`, which is defined just above.